packages/salesforcedx-vscode-apex/src/embeddedSoql/soqlCompletion.ts

```diff
@@ -145,5 +146,4 @@ function eolForDocument(doc: TextDocument) {
     case EndOfLine.CRLF:
       return '\r\n';
   }
-  return '\n';
```

Comment:
show we throw here verse returning undefined where we use to default to a value?

Reply:
I'll just restore the default. Not sure why I nuked it.